tests/test_calendar_privacy.py

```diff
@@ -0,0 +1,54 @@
+from app.internal.calendar_privacy import can_show_calendar
+# after user system is merged: from app.internal.security.dependancies import CurrentUser
+from app.routers.user import create_user
+
+import pytest
+
+
+USER_DATA = {
+    'username': 'new_test_username',
+    'email': 'new_test.email@gmail.com',
+    'password': 'new_test_password',
+    'language': 'english'
+}
+
+
+def test_can_show_calendar_public(session):
+    user = create_user(
+            session=session,
+            **USER_DATA
+    )
+    user.privacy = "Public"
+    #  to be replaced after user system is merged:current_user = CurrentUser(**user.__dict__)
+    current_user = user
+    result = can_show_calendar(requested_user_username='new_test_username', db=session, current_user=current_user)
+    assert result
```

Comment:
`assert 4` is also true.

Reply:
Not sure what you mean?
Result can be only True or None .. how can it ever be 4?

Reply:
I meant you need to be more specific here - `assert result is True`, since `assert 4` won't fail (you want to check that result is True here, right?)

Reply:
I'm not entirely sure if this is best practice but I implemented itso hopefully Yam will be fine with this :)